.vsts-ci.yml

```diff
@@ -35,17 +35,18 @@ jobs:
     matrix:
       regular:
         TASK: regular
+        PYTHON_VERSION: '3.9'
```

Comment:
I tried to use different Python versions for the same `TASK` in `Linux` and `Linux_latest` jobs. The same is true for macOS configurations on Azure Pipelines and GitHub Actions.